website/docs/reference/spicepod/keywords.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Reserved Keywords'
+sidebar_label: 'Reserved Keywords'
+description: 'Reserved keywords for datasets'
+tags:
+  - reference
+  - spicepod
+  - datasets
+---
+
+The following keywords cannot be used as names for datasets.
+
+## General Protected Keywords
+
+These keywords apply to all data connectors:
+
+- `COUNT`
+- `FALSE`
+- `NULL`
+- `END-EXEC`
+- `LATERAL`
+- `TABLE`
+- `UNNEST`
+
+## Connector-Specific Protected Keywords
+
+### ClickHouse
```

Comment:
These should go into each individual connectors docs (and perhaps linked from here).